homeassistant/components/withings/__init__.py

```diff
@@ -0,0 +1,136 @@
+"""
+Support for the Withings API.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/withings/
+"""
+import voluptuous as vol
+from homeassistant.config_entries import ConfigEntry
```

Comment:
Please group and sort imports according to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports).

Ie, add a blank line here.